### PR TITLE
Feature/khb gallery in authority detail

### PR DESCRIPTION
**Pull request by igor-kamil**
# Description

Display gallery in author detail.
Using plugins:

- https://isotope.metafizzy.co/ (for masonry layout)
- http://dimsemenov.com/plugins/magnific-popup/ (for popup functionality with gallery)

Fixes WEBUMENIA-872

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/views/autor.blade.php [hunk 1]
```diff
@@ -131,59 +160,53 @@
...
 @stop
 
-
 @section('javascript')
-{!! Html::script('js/readmore.min.js') !!}
-
-{!! Html::script('js/slick.js') !!}
-{!! Html::script('js/components/artwork_carousel.js') !!}
+<script src="https://unpkg.com/isotope-layout@3/dist/isotope.pkgd.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/1.1.0/jquery.magnific-popup.min.js"></script>
```
Comment (phivk) [inline resources/views/autor.blade.php]: would it be preferable to have these files locally?

File: resources/views/autor.blade.php [hunk 2]
```diff
@@ -77,12 +77,41 @@
...
                         'show' => true,
                     ])
-                    @include('components.khb_accordion_card', [
-                        'title' => utrans('autor.gallery'),
-                        'content' => '',
-                        'parrentId' => 'authorAccordion',
-                        'show' => true,
-                    ])
+
+                    {{-- gallery --}}
```
Comment (igor-kamil) [inline resources/views/autor.blade.php]: @phivk sorry I know this is not nice. would be better to do it somehow with combination of @yield directive... but that's probably for later

File: resources/views/layouts/master.blade.php [hunk 3]
```diff
@@ -54,6 +54,8 @@
     </div>
 
     <!-- Javascript -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/4.1.1/lazysizes.min.js"></script>
```
Comment (phivk) [inline resources/views/layouts/master.blade.php]: same question here re local vs CDN...?

File: resources/views/autor.blade.php (hunk 1 above)
Comment (igor-kamil) [reply resources/views/autor.blade.php]: if we can use CDN - it's better to use CDN

https://www.sitepoint.com/7-reasons-to-use-a-cdn/
https://webmasters.stackexchange.com/questions/92083/local-files-vs-cdn

it really depends. usually each plugin has its own requirements and it's nice to let it be solved by npm. but it's not necessary in this case. and we don't customize these files.

**Pull request merged**